install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
+configured automatically. However, iptables rules can break {product-title} if
+not configured correctly. The advantages of firewalld include allowing multiple
+objects to safely share the firewall rules. However, it is not available on RHEL
+Atomic Host.
+
+To use firewalld as the firewall for an {product-title} install, add the
+`os_firewall_use_firewalld` variable to the list of configuration variables in
+Ansible host file on install:
+
+----
+[OSEv3:vars]
+os_firewall_use_firewalld=True
+----
+
```

Comment:
@knobunc @sdodson Not sure about this one either. If it's not the default, then why recommend it? I'll see what the response above is.